web_widget_mail_send_odoo/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Web Widget - Email internal mail link
+=====================================
+
+This module was written to provide a new widget for Email fields to replace the
+standard behaviour of displaying a 'mailto:' link with JS to open the internal
+Email wizard from Odoo for message composition.
+
+The message you sent gets appended to the chatter as if you where using internal
```

Comment:
I thought this is past: 'I sent you the message 2 hrs ago' vs. present: 'I send you a message in just a minute'. I think it's correct, but I'm not a native speaker.  
The message gets appended to the chatter AFTER it has been sent.